main.js

```diff
@@ -178,9 +182,39 @@ function logActivity() {
      }
  }
    saveCard();
+   clearTimerSection();
```

Comment:
not sure why cleartimerSection() is being called inside this saveCare() function

Reply:
Never mind, I read this wrong